test/e2e/examples.go

```diff
@@ -32,8 +32,8 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/testfiles"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"github.com/onsi/ginkgo"
```

Comment:
I don't think we want this.

https://onsi.github.io/ginkgo/#avoiding-dot-imports is maybe another way to address this, but I'm not sure whether it's worth the effort.

Reply:
@ixdy Thanks for your feedback. Sorry for the delay.
But golint failed. Do we have a better way to make golint go through now?